libr/cons/dietline.c

```diff
@@ -442,7 +442,112 @@ R_API int r_line_hist_chop(const char *file, int limit) {
 	return 0;
 }
 
+static void draw_selection_widget () {
```

Comment:
all these functions, according to the style, should have `(` without a space before it.